engine/eConstr.mli

```diff
@@ -321,6 +321,8 @@ val of_rel_decl : (Constr.t, Constr.types) Context.Rel.Declaration.pt -> (t, typ
 
 val to_rel_decl : Evd.evar_map -> (t, types) Context.Rel.Declaration.pt -> (Constr.t, Constr.types) Context.Rel.Declaration.pt
 
+val map_to_rel_context : (t -> Constr.t) -> (t, types) Context.Rel.pt -> (Constr.t, Constr.types) Context.Rel.pt
```

Comment:
I am not sure it is a great idea to export this specialized function. You should rather convert it and map, or the other way around, efficiency shouldn't matter here. Otherwise we'll end up with a bloated API.

Reply:
Sure whatever